main.go

```diff
@@ -120,16 +120,22 @@ func httpJob(ctx context.Context, args JobArgs) error {
 		for key, value := range jobConfig.Headers {
 			req.Header.Add(parseStringTemplate(key), parseStringTemplate(value))
 		}
+
+		startedAt := time.Now().Unix()
+		log.Printf("[DEBUG] %s %s started at %d", jobConfig.Method, jobConfig.Path, startedAt)
```

Comment:
It would be ideal to use some more sophisticated logging library that supports logging levels so that we could only include debug logging when needed. Can be done in a separate PR if you want